ui/src/app/shared/components/orb/policy/policy-datasets/policy-datasets.component.html

```diff
@@ -39,7 +39,7 @@
 </ng-template>
 
 <ng-template #groupTemplateCell let-i="index" let-row="row" let-value="value">
-    <button (click)="onOpenGroupDetails(row?.agent_group)"
```

Comment:
I saw you created new function to perform the action.. Did you removed this **onOpenGroupDetails** function or still being used?

Reply:
yes, I strongly believe we removing code that are not being used would be good to don't confuse other programmers that can be working on the code futurely.